ships/0040-build-runtime-class.md

```diff
@@ -0,0 +1,159 @@
+<!--
+Copyright The Shipwright Contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+---
+title: build-runtime-class
+authors:
+  - "@adambkaplan"
+reviewers:
+  - "@dorzel"
+  - "@HeavyWombat"
+approvers:
+  - "@qu1queee"
+  - "@SaschaSchwarze0"
+creation-date: 2025-03-25
+last-updated: 2025-03-25
+status: provisional
+see-also:
+  - "/ships/0039-build-scheduler-opts.md"  
+replaces: []
+superseded-by: []
+---
+
+# Build Runtime Class
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [docs](/docs/)
+
+## Open Questions [optional]
+
+- Should user namespaces also be in scope for this feature?
```

Comment:
This is still a Kubernetes Beta feature which is disabled by default afaik. Anyway, I would be okay.

Reply:
I decided to move user namespaces out of scope. It is a bit odd that it is "beta" but still off by default. I guess not all CRI runtimes support it (and something like KinD may not be able to ever support it?).